src/platform/src/core/utils/secureApiProxyClient.js

```
@@ -74,19 +93,42 @@ const createSecureApiClient = () => {
+          // No authentication needed
+          break;
+        }
+        case AUTH_TYPES.API_TOKEN: {
+          // Use API token from environment variables
+          const apiToken = getApiToken();
+          if (apiToken) {
+            config.headers['Authorization'] = `Bearer ${apiToken}`;
+          }
+          break;
+        }
+        case AUTH_TYPES.JWT:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)